packages/demux-icon/src/IconActionReader.ts

```diff
@@ -76,7 +80,7 @@ export class IconActionReader extends AbstractActionReader {
 
       return irreversibleBlockNum;
     } catch (err) {
-      this.log.error(err);
+      // this.log.error(err);
```

Comment:
@boomspeed94 I don't see any problem with this error log. Please do not comment it out.

Reply:
@manh-vv  reverted